tests/requirements.txt

```diff
@@ -15,6 +15,6 @@ coverage>=4.5.2
 # Optional Testing Dependencies
 nltk>=3.2
 # spacy>=2.0.18
-pandas>=0.20
+pandas>=0.20,<1
```

Comment:
Thanks for the intermediate fix on this! Sounds like we'll take a closer look at this in #1016